src/atoms.jl

```diff
@@ -51,6 +51,27 @@ _read_convert(value::Int32) = Int(value)
 _read_convert(value::Vector{Int32}) = Int.(value)
 
 _write_convert(value) = value
+_write_convert(value::T) where {T<:Unitful.Length} = ustrip(uconvert(u"Å", value))
+_write_convert(value::Vector{T}) where {T<:Unitful.Length} = ustrip.(uconvert.(u"Å", value)) 
```

Comment:
I agree that all of these routines wouldn't be needed in the case by calling it `_write_convert.(value)`.
This should work, but i don't know how to call it in (l. 83):
`_dict_remap_rev(d) = _dict_remap_names(d, REV_NAME_MAP, string, String, _write_convert)`

I even thought about adding a routine for a Matrix, because yet there is no possibility to use units with a Matrix (just with Vector{Vector{T}}, but i agree, if one calls it with `_write_convert.(value)` there should be no need for all of these routines.

Reply:
I think those two routines are needed, because otherwise the hcat((value)...) part would be lost in the case it is needed.

Reply:
OK, good solution